docs/installation/index.md

```diff
@@ -12,3 +12,5 @@ The following sections detail how to set up a new instance of NetBox:
 If you are upgrading from an existing installation, please consult the [upgrading guide](upgrading.md).
 
 NetBox v2.5 and later requires Python 3.5 or higher. Please see the instructions for [migrating to Python 3](migrating-to-python3.md) if you are still using Python 2.
+
+Netbox v2.5.7 and later moved to using systemd instead of supervisord.  Please see the instructions for [migrating to systemd](migrating-to-systemd.md) if you are still using supervisord.
```

Comment:
`v2.5.7` should be updated to the version number when it is released, 2.5.8 it is currently anyway

Reply:
Updated it to 2.5.9 which would be the next version release.  There were a few others as well.

Reply:
@DanSheps I guess you have to update this one again :P

Reply:
Yeah, I will wait until we figure out what the plan is though.